### ISSUE TITLE
WTHIT not found on server, despite being on server

### ISSUE THREAD

**Issue by katubug** [badasintended/wthit#67]
### Describe the issue

Hi! I'm troubleshooting something else, but this was in my log:

`[10:30:46] [Render thread/WARN]: WTHIT is not found on the server, all syncable config will reset to their default value.`

I do have WTHIT on both client and server, though. This is Fabric 0.11.6 and WTHIT 3.8.1

Thanks for your time!

### Log output

https://pastebin.com/6YKXb5WA

**Comment by deirn** [comment]
What version of WTHIT the server has?

**Comment by katubug** [comment]
Sorry for the delayed response! 3.8.1 is the version on the server.

**Comment by deirn** [comment]
Hmm, say, if you have a container (eg. chest) with a custom name and look into it, does WTHIT show the custom name?

**Comment by katubug** [comment]
Yes. Just renamed a chest and placed it down, and when opening the
inventory, it shows the name on the top of the gui.

**Comment by deirn** [comment]
No, I mean the WTHIT tooltip, does it show the custom name?

**Comment by katubug** [comment]
Oh gotcha. Yes, it does show the name in the tooltip.

**Comment by deirn** [comment]
Huh, do you happen to use a proxy server like Bungeecord?

**Comment by katubug** [comment]
I don't think so. I use Pterodactyl panel on a VDS. Other than that, just
fabric.

**Comment by deirn** [comment]
Oh well, can you show me your server log?

**Comment by katubug** [comment]
Sorry for the delayed response. The server log is attached to the original post.

**Comment by deirn** [comment]
> `[Render thread/INFO]`

Idk, that looks like a client log to me :p

**Comment by katubug** [comment]
Oof, my memory is awful so you're probably right. Honestly though, I think
it's just a false error because it seems to be working just fine. If you
still want the log, I will post it next time I get to my computer.
Otherwise, feel free to close this. I'm sorry, I feel as if I've wasted
your time.
